materialized_sql_view/model/abstract_materialized_sql_view.py

```diff
@@ -336,9 +336,12 @@ def create_mat_view(self, cr, sql, view_name, mat_view_name):
                    dict(mat_view_name=mat_view_name,
                         view_name=view_name,
                         ))
+        cr.execute("CREATE UNIQUE INDEX %(mat_view_name)s_index "
+                   "ON %(mat_view_name)s (id)" %
+                   dict(mat_view_name=mat_view_name, ))
 
     def refresh_mat_view(self, cr, view_name, mat_view_name):
-        cr.execute("REFRESH MATERIALIZED VIEW %(mat_view_name)s" %
+        cr.execute("REFRESH MATERIALIZED VIEW CONCURRENTLY %(mat_view_name)s" %
```

Comment:
Hi,

Thanks for this PR, ``CONCURRENTLY`` option is available from PG9.4 only not under PG9.3 so I would suggest to create a new class called PG090400 to implement this feature and adapt getInstance method.